modules/search/client/search.client.module.js

```diff
@@ -1,3 +1,22 @@
-// Use application configuration module to register a new module
-// The core module is required for special route handling; see /core/client/config/core.client.routes
+import AppConfig from '@/modules/core/client/app/config';
+
+import '@/modules/core/client/core.client.module';
+
 AppConfig.registerModule('search', ['core']);
+
+// config
```

Comment:
I like how these things become explicit...

Reply:
Yes, it's either ugly and verbose, or explicit and clear, depending on perspective.